bin/heudiconv

```diff
@@ -238,14 +231,25 @@ def embed_nifti(dcmfiles, niftifile, infofile, force=False):
     #orig_nii.update_header()
     #orig_nii.to_filename(niftifile)
     meta = ds.NiftiWrapper(new_nii).meta_ext.to_json()
-    with open(infofile, 'wt') as fp:
+    with open(infofile, 'at') as fp:
```

Comment:
have you tested this with and without the bids flag? and in both cases all the other dicom info is there?